com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/tester.py

```diff
@@ -424,7 +422,9 @@ def test(self, ctxtype, config=None, assert_on_fail=True, username=None, passwor
             config: Configuration for submission.
             assert_on_fail(bool): True to raise an assertion if the test fails, False to return the passed status.
             username(str): username for distributed tests
+                .. deprecated:: 1.8.3
```

Comment:
Need an indication of what a deprecated item is being replaced with.

Reply:
Added doc indicating environment variables should be used.